deployments/epinio.go

```diff
@@ -165,7 +165,7 @@ func (k Epinio) apply(ctx context.Context, c *kubernetes.Cluster, ui *termui.UI,
 				strings.Contains(err.Error(), "EOF")
```

Comment:
There is one more error we should retry on here. Since you are at it maybe you can add it (or we can create an issue for it):

```
#1: creation of production ssl certificate failed: conversion webhook for cert-manager.io/v1alpha2, Kind=Certificate failed: Post "https://cert-manager-webhook.cert-manager.svc:443/convert?timeout=30s": x509: certificate signed by unknown authority
```

It happens on a public cloud provider the first time you install Epinio. If you uninstall and retry it usually works. Must have something to do with the fact that the first time, images have to be downloaded making things slower, thus we hit this error. I don't want to hijack this PR though, I can move it to another issue.

Reply:
Ah, that is the epinio deployment. Touched only because disliked the message ... I will add a `x509` pattern as I have in my retry.

Reply:
Actually, might be better to replace `failed calling webhook` with the `x509`. That last should call anything certs, regardless of webhook or not.

Reply:
Done with commits b9ad829867534bab1224e007655c7c8b61b8040b, f4ade8929dbf718b249526880f2374efbb77eb50, and c772e169b2c6779124cc7279ab13c274b76139b4.